### ISSUE TITLE
What US  import/ export most  in 2018?

### ISSUE THREAD

**Issue by collleenwang** [jsoma/data-studio-projects#211]
# Pitch

The U.S. is a vast country, and everyday international trade flows go and back. What kind of items the U.S. need to import/export most? 

And if possible to ask two follow-up questions,(1) what these items can influence the U.S.? Can this influence be quantitative?   (2) What about that in 2017? Comparison? 

## Summary

After seeing too many dizzy items, let's make things simple.

## Details

**Possible headline(s): What US  import/ export most in international trade flows?

**Data set(s):**  I download a bunch of data from census gov. https://www.census.gov/foreign-trade/data/index.html. And I will explore the data in the Q1 of 2018, and maybe in recent 2-3 years also.

**Code repository:** 
	
**Possible problems/fears/questions:** 
- The data is not very completed, so I just use what I can find and take advantage to see.

- In the raw xlsx files, under time(month) category, it splits three columns: balance, export, and import. Should I sum them together? Or any other better way? I have to make them easier to make my analysis with tools.   

-I think it will be better way to show an interactive map, due to the data insights are clear and implicit, so I want to make something funny, but I'm still figuring it out. So let's see what can I do so far.

## Work so far

![screen shot 2018-07-24 at 05 11 17](https://user-images.githubusercontent.com/39416429/43128475-149685c4-8f00-11e8-8e08-3e5c7f8946ca.png)

Let's see detailed categories.So in tech, we can these ranks. 

![screen shot 2018-07-24 at 05 14 29](https://user-images.githubusercontent.com/39416429/43128658-829d129a-8f00-11e8-9841-81bbda8888f9.png)

Oh, really? 

![screen shot 2018-07-24 at 05 25 28](https://user-images.githubusercontent.com/39416429/43129293-1481ad78-8f02-11e8-8b61-309f9d8bf4b0.png)

Ok, it makes sense, China should export many tech manufacture products to US.

![screen shot 2018-07-24 at 05 30 54](https://user-images.githubusercontent.com/39416429/43129670-f6f03f1c-8f02-11e8-87bf-8cf980afaa9d.png)

Feel confused again!.....Is the data correct?

![screen shot 2018-07-24 at 06 15 02](https://user-images.githubusercontent.com/39416429/43132226-14b9b13a-8f09-11e8-9904-0322f9886c0c.png)
Ok, let's see other categories, but just the balance, no other further info.

![screen shot 2018-07-24 at 06 29 51](https://user-images.githubusercontent.com/39416429/43132850-0dfed530-8f0b-11e8-91dc-44a10865b737.png)

![screen shot 2018-07-24 at 06 36 30](https://user-images.githubusercontent.com/39416429/43133197-ffebd3fc-8f0b-11e8-9a4d-dd585af5cf6d.png)

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [x] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [ ] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [ ] My issue includes images - either inspiration or what I've done so far

**Comment by ella24** [comment]
Hello!
I noticed that you do not have your code's repository link in here. I would like for you to add it because the dfs are always useful when giving feedback. 
I recommend doing a graph of exports and imports per country. You already have the countries. But it will be great to see what stuff they are buying or sending. 
In the first four graphs you already have the quantity and the end of the bar, there is no need for the x-axis to have numbers(again). 
I do not understand the different colors in the next to last and last graph. Is it the sum value too? if it is, you should add it in the side.
There is also so many stuff. Maybe use the top ten foods and capital goods?
With what are you planning to compare this outcomes? will you do it per year? or something else? Looking forward to see the rest! 
This might interest you: https://www.forbes.com/sites/niallmccarthy/2018/07/12/what-the-american-public-thinks-about-u-s-trade-relations-infographic/#692c86df4556

**Comment by benbitoun** [comment]
I agree with the feedback of Coral. So many stuff. And you gave the answer yourself: You're thinking that interactive would be better so it's on the readers to choose something interesting. But I'm stressed and tired and would rather have you as a journalist do that choosing for me and tell me an interesting story. So I would look at the data and the graphs and focus on something, tell me why it's important and make something nice out of it.

**Comment by jessimckenzi** [comment]
I think this is a timely and interesting idea, especially with all of the news about different tariffs and trade wars and so on.

On the graphs and charts above: the colors seem random, and if the dark colors mean more things and the light colors mean less (in the box chart) what does the size of the box mean??

It's not clear what interests you about this data set, and frankly I'm not interested in the dataset as a whole, yet. I think you still need to do a lot of whittling down and then to TELL ME what's interesting about it. The more narrow it is, the more idiosyncratic things will stick out. See what Paiva did with the tomato market: :https://palarisk.github.io/tomatoes/; https://github.com/jsoma/data-studio-projects/issues/123

I wasn't interested in tomatoes before, but wow it is weird and interesting that there's so many grown in China! Who knew? I think you can do something similar here. My suggestion: I think it's p interesting that we export so much aerospace technology. But how can you learn more? Are we a leader in aerospace tech? Comparing it to communications tech or biotech is not as interesting to me as finding more meaning from another dataset or comparing it to other countries' aerospace exports.

**Comment by sarahslo** [comment]
here i'd like you to organize the data by the data and not alphabetically. when you have charts that look this noisy, the way to start to understand what they tell you is to organize it by the data. once you have it sorted A-Z, least to most you and see the biggest and the smallest. and you can start to make sense of it. from there you can divide it into top or bottom. 

what is the value on the export tree maps? i don't know what the number represents.
